packages/core/types/index.d.ts

```diff
@@ -80,11 +80,12 @@ declare module '@boisu/core' {
     export function addDirectVideoOutputSink(sinkId: string): void;
     export function removeDirectVideoOutputSink(sinkId: string): void;
     export function clearVideoOutputSink(streamId: string): void;
-    // TODO: what is something
+    // If onFrame is not passed, this function will clear the sink
+    // TODO: what is "something" - Discord always passes true
     export function setVideoOutputSink(
         streamId: string,
-        onFrame: FrameCallback,
-        something: boolean,
+        onFrame?: FrameCallback,
+        something?: boolean,
```

Comment:
```suggestion
        onFrame: FrameCallback,
        something: boolean,
```
This isnt needed because this only gets called by the `engine` counterpart. It's kinda cringe imo so I made `clearVideoOutputSink` for that